### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by Bhuvaneshwari29**
Completed my Javascript1 week2 homework

### CODE REVIEW

File: javascript/javascript1/week2/jsweek2hw.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+     return firstname+' '+surname;
+ }
+}
+const fullName3 = getFullName('Bhuvaneshwari','Periyasundaram',true);
+const fullName4 = getFullName('Sanju','sri');
+console.log(fullName3);
+console.log(fullName4);
+
+//Event application
+const week = ['sunday','monday','tuesday','wednesday','thursday','friday','saturday','sunday'];
```
Comment (florinmaticu) [inline javascript/javascript1/week2/jsweek2hw.js:32]: you have added sunday two times
```suggestion
const week = ['sunday','monday','tuesday','wednesday','thursday','friday','saturday'];
```

File: javascript/javascript1/week2/jsweek2hw.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  }
+  else if(class07Students.includes(studentName))
+  {
+      console.log('Student'+studentName+'is already in the class');
+  }
+  else if(studentName==='Queen')
+  {
+      class07Students.push(studentName);
+  }
+  else if(class07Students.lenght>=6)
```
Comment (florinmaticu) [inline javascript/javascript1/week2/jsweek2hw.js:79]: 6 is still okay
```suggestion
  else if(class07Students.lenght>6)
```

File: javascript/javascript1/week2/jsweek2hw.js (hunk 1 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript1/week2/jsweek2hw.js]: Removed the extra array input Sunday

File: javascript/javascript1/week2/jsweek2hw.js (hunk 2 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript1/week2/jsweek2hw.js]: Changed to > from >=

**Review by florinmaticu — APPROVED** [review]

**Pull request merged**